veps/sig-network/hotpluggable-nad-ref.md

```diff
@@ -0,0 +1,194 @@
+# VEP #140: Live Update NetworkAttachmentDefinition Reference
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [X] (R) Target version is explicitly mentioned and approved
+- [X] (R) Graduation criteria filled
+
+## Overview
+
+VMs using secondary networks refer to a NetworkAttachmentDefinition object's namespace and name as part of their network specification. 
+Currently, the whole network specification, including this reference - is immutable for the lifecycle of a VM.
+If a user wishes to change the network a VM is connected to (example: switching VLANs), they must modify the VM specification and restart the VM. 
+This proposal introduces the ability to update the NAD reference (networkName) of a network on a running VM through Live Migration. 
```

Comment:
I would take this a step further and indicate why this "small" change means so much - restart impacts the workload, while live migration is transparent to the guest.